packages/flutter_tools/lib/src/version.dart

```diff
@@ -158,12 +187,12 @@ class FlutterVersion {
   /// Return the branch name.
   ///
   /// If [whitelistBranchName] is true and the branch is unknown,
-  /// the branch name will be returned as 'dev'.
+  /// the branch name will be returned as '[user-branch]'.
   String getBranchName({ bool whitelistBranchName: false }) {
```

Comment:
`whitelistBranchName` seems like a weird name for this argument... it's more like `redactUnknownBranches`

Reply:
Yeah. Fixed.